google-cloud-spanner/src/main/java/com/google/cloud/spanner/connection/UnitOfWork.java

```diff
@@ -39,9 +41,11 @@
 
   enum UnitOfWorkState {
     STARTED,
+    COMMITTING,
     COMMITTED,
     COMMIT_FAILED,
     ROLLED_BACK,
+    RUNNING,
```

Comment:
This state is added to indicate that a batch is running, but not yet finished, and that all subsequent statements should not be included in the batch.